extensions/amp-dynamic-css-classes/amp-dynamic-css-classes.md

```diff
@@ -35,3 +35,14 @@ classes: `amp-referrer-www-google-com`, `amp-referrer-google-com`, and
 
 The `amp-viewer` class will be set if the current document is being
 displayed inside a Viewer.
+
+**amp-ua-***
+
+A User Agent class will be added when we detect that the document is
```

Comment:
Is https://github.com/ampproject/amphtml/blob/master/extensions/amp-dynamic-css-classes/amp-dynamic-css-classes.md#behavior not enough?